README.md

```diff
@@ -19,9 +19,17 @@ ShortcutJunkie is a MEAN app built off of the mean.js boilerplate. The stack is
 * `public` – Public files for the application
     * `dist` – All the files that are actually delivered to the client side. These are compiled by the build system.
     * `lib` – Front-end dependencies are installed here by bower. They are read from here in in the development and test environments, but they are concatenated and minified into `dist` for production.
-* `uploads` – This was created by mean.js, and I'm not sure why. We'll probably get rid of it.
+* `uploads` – This was created by MEAN.JS, and I'm not sure why. We'll probably get rid of it.
 
-For more information about how the code is structured, you can refer to the [mean.js](http://meanjs.org/docs.html#folder-structure) docs. Unfortunately they are a little outdated since ShortcutJunkie is built off a newer version of the project, but they can give you a good idea of where you should look for certain functionality.
+For more information about how the code is structured, you can refer to the [MEAN.JS](http://meanjs.org/docs.html#folder-structure) docs. Unfortunately they are a little outdated since ShortcutJunkie is built off a newer version of the project, but they can give you a good idea of where you should look for certain functionality.
+
+#Contributing
+Contributions to ShortcutJunkie are welcome. To setup your local development environment, you'll need to do the following.
+* Follow the [Prerequisite instructions](http://meanjs.org/docs.html#getting-started) for MEAN.JS.
+* Optional: Follow the [Downloading instructions](http://meanjs.org/docs.html#getting-started) for installing MEAN.JS. This is only needed if you want to add new modules to the ShortcutJunkie project.
+* Install project dependencies with `npm install`
+* Install gulp with `npm install -g gulp`
```

Comment:
It's weird that gulp doesn't get installed with `npm install` since it's in the default dependencies. Better safe than sorry though!